finder-app/finder.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Accept 2 runtime arguments
+if [ $# -ne 2 ]; then
+  echo "Expected number of args is 2, not [$#]."
```

Comment:
@qrogers6 I like the idea to put an error message here but how could you make it more helpful for the user?